src/options.rs

```diff
@@ -223,16 +223,14 @@ pub fn parse_args() -> Mode {
         } else {
             BackgroundColor::Dark
         }
-    } else {
-        if let Ok(background) = env::var("DFT_BACKGROUND") {
-            if background == "light" {
-                BackgroundColor::Light
-            } else {
-                BackgroundColor::Dark
-            }
+    } else if let Ok(background) = env::var("DFT_BACKGROUND") {
```

Comment:
It looks like this logic could be simplified even further:

```suggestion
    } else if let Ok("light") = env::var("DFT_BACKGROUND").as_deref() {
        BackgroundColor::Light
    } else {
        BackgroundColor::Dark
    }
```

Reply:
I worry a little that it makes things a little less clear: right now it reads:

"If `matches` specifies background, use that. If not, and the environment variable is set, use that. Otherwise, use a default", but with that change, it reads as

"If `matches` specifies background, use that. If not, and the environment variable is set to "light", make it light, otherwise, use the default".

It's subtly different, and right now, they actually mean the same thing, but might be a surprise if another fallback is added before the default. I actually almost made the mistake of trying to apply the same change to the original `if`, but that would have actually changed behavior (if matches specified it, but dark, it could be overridden by the environment variable)

Still happy to make the change if you think it's better.